test/support/live_views/elements.ex

```diff
@@ -49,7 +49,7 @@ defmodule Phoenix.LiveViewTest.ElementsLive do
 
     <%# hooks %>
     <section phx-hook="Example" id="hook-section" phx-value-foo="ignore">Section</section>
-    <section phx-hook="Example" class="idless-hook">Section</section>
+    <section phx-hook="Example" id="hook-section-2" class="idless-hook">Section</section>
```

Comment:
had to add ID here. It means the validation is working 🎉

Reply:
NICE!